bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -7,27 +7,14 @@ The Netatmo binding integrates the following Netatmo products:
 - *Indoor Camera / Welcome*. Reports last event and persons at home, consult picture and video from event/camera.
 - *Outdoor Camera / Presence*. Reports last event, consult picture and video from event/camera.
 
-See https://www.netatmo.com/ for details on their product.
-
-Please note, recent Netatmo thermostats are not supported because they require the Energy API which is not yet implemented in the binding.
-Only older Netatmo thermostats compatible with the Thermostat API are supported.
-For the same reason, Netatmo valves are also not supported.
-
+See http://www.netatmo.com/ for details on their product.
```

Comment:
Why changed from https to http? I would propose to change it back.

Reply:
Yes, changed